pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -158,13 +158,16 @@ func (s *ServiceImpl) getServerAdminNode(c *models.ReqContext) *navtree.NavLink
 	}
 
 	adminNode := &navtree.NavLink{
-		Text:        "Server admin",
-		Description: "Manage server-wide settings and access to resources such as organizations, users, and licenses",
-		Id:          navtree.NavIDAdmin,
-		Icon:        "shield",
-		SortWeight:  navtree.WeightAdmin,
-		Section:     navtree.NavSectionConfig,
-		Children:    adminNavLinks,
+		Text:       "Server admin",
+		Id:         navtree.NavIDAdmin,
+		Icon:       "shield",
+		SortWeight: navtree.WeightAdmin,
+		Section:    navtree.NavSectionConfig,
+		Children:   adminNavLinks,
+	}
+
+	if s.cfg.IsFeatureToggleEnabled(featuremgmt.FlagTopnav) {
```

Comment:
this is so we don't show the subtitle in the hover menu when topnav is off:
![image](https://user-images.githubusercontent.com/20999846/193531338-2dd2109d-ed63-446b-835e-f3e1d0a83e3e.png)